tracing-journald/src/lib.rs

```diff
@@ -155,6 +157,31 @@ impl Subscriber {
         &self.syslog_identifier
     }
 
+    /// Sets the syslog facility for this logger.
+    ///
+    /// The syslog facility comes from the classic syslog interface (`openlog()`
+    /// and `syslog()`). In syslog the facility argument is used to specify
```

Comment:
tiny, unimportant nit: since these are libc functions in `man 3`, should we refer to them like this?
```suggestion
    /// The syslog facility comes from the classic syslog interface (`openlog(3`
    /// and `syslog(3)`). In syslog the facility argument is used to specify
```

Reply:
This comment was mostly copy & paste from `Layer::with_syslog_identifier()` so same change would be worth applying to both. I can do that.

Reply:
Fixed.